events/events.c

```diff
@@ -170,17 +182,17 @@ events_run(void)
 
 /**
  * events_spin(done):
- * Run events until ${done} is non-zero (and return 0), an error occurs (and
+ * Call events_run until ${done} is non-zero (and return 0), an error occurs (and
  * return -1), or a callback returns a non-zero status (and return the status
- * code from the callback).
+ * code from the callback).  May be interrupted by events_interrupt.
```

Comment:
should this be listed before the `(and return 0)` clause?  I tried moving it there, but it made the sentence seem a bit awkward.